### ISSUE TITLE
New reward function

### ISSUE THREAD

**Issue by bigaidream** [bigaidream-projects/qan#26]
Change our current reward function to the one used in *Learning Step Size Controllers for Robust Neural Network Training*. 

In that paper, the reward considers all the previous training loss reduction in logarithm form. This might make the actions to change less frequently.

**Comment by bigaidream** [comment]
Done.
